app/src/main/java/com/kylecombes/micrepeater/WelcomeActivity.java

```diff
@@ -34,37 +23,48 @@ protected void onCreate(Bundle savedInstanceState) {
         pagerAdapter = new ScreenSlidePagerAdapter(this);
         viewPager.setAdapter(pagerAdapter);
 
-        final Button backbutton = (Button) findViewById(R.id.back_button);
-        final Button nextbutton = (Button) findViewById(R.id.next_button);
-        backbutton.setOnClickListener(new View.OnClickListener() {
+        final Button backButton = findViewById(R.id.back_button);
+        final Button nextButton = findViewById(R.id.next_button);
```

Comment:
Can we namespace these a bit more please? Like `R.id.welcome_wizard_back_button` (and likewise for `next_button`).

Reply:
Fixed